src/components/Install/index.js

```diff
@@ -22,7 +22,7 @@ export default function Install() {
               alt="1. "
               src={require('/docs/assets/counting-01.png').default}
               ></img>
-            <a className="link" href="docs/guide/installation/prerequisites#compute">
+            <a className="link" href="https://llm-d.ai/docs/guide/installation/prerequisites#compute">
```

Comment:
I don't like having to add the domain here, but it looks like the relative link here doesn't do what we want.

Reply:
Hrm - does it work with the `/` in there like:
```<a className="link" href="/docs/guide/installation/prerequisites#compute">```